aten/src/ATen/CMakeLists.txt

```diff
@@ -178,17 +178,21 @@ file(GLOB native_flash_attn_api_cpp "native/transformers/cuda/flash_attn/flash_a
 file(GLOB flash_attention_hip_hip "native/transformers/hip/flash_attn/*.hip")
 # if USE_FLASH_ATTENTION is set, ensure CK instances get generated
 if(USE_FLASH_ATTENTION)
-  if(DEFINED ENV{USE_CK_FLASH_ATTENTION})
+  if(DEFINED ENV{USE_ROCM_CK_SDPA} OR DEFINED ENV{USE_CK_FLASH_ATTENTION})
```

Comment:
Please make `USE_ROCM_CK_SDPA` a CMake option in `CMakeLists.txt` and have it be always set to `False` so we don't have to query the env var here, and can directly conditionalize on the CMake flag. It'll make no difference on the user side, because they can still set an env var `USE_ROCM_CK_SDPA` which will set the corresponding CMake variable due to https://github.com/pytorch/pytorch/blob/48fe4ff2479d704e67f274969aa18bcccbe7a8dd/tools/setup_helpers/cmake.py#L317